assets/scss/variables.scss

```diff
@@ -6,6 +6,7 @@ $white: #fff;
 $grey: #777;
 $offwhite: #f1f1ee;
 $paper: #d9d4ca;
+$lightmagenta: #e7e7e9;
```

Comment:
I wanted to use a more descriptive name than `someothershadeofgrey` since we already have several shades of grey in the variables, I googled the hexcode and #e7e7e9 is a type of magenta https://encycolorpedia.com/e7e7e9

Reply:
let's go for lightbluemagenta then

Reply:
ok updated